boards/arm/arduino_portenta_h7/board.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2022 Benjamin Björnsson <benjamin.bjornsson@gmail.com>.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <init.h>
+#include <drivers/gpio.h>
+
+static int board_init(const struct device *dev)
+{
+	const struct gpio_dt_spec led1 = GPIO_DT_SPEC_GET(DT_ALIAS(led1), gpios);
+
+	ARG_UNUSED(dev);
+
+	if (!device_is_ready(led1.port)) {
+		return -ENODEV;
+	}
+
+	return gpio_pin_configure_dt(&led1, GPIO_OUTPUT_INACTIVE);
```

Comment:
I've chosen to turn led1 off here since I've experienced that the Arduino bootloader leaves it on.
Arduino does the same in the variantInit function found in variants.cpp.

Reply:
Ok, then please add a comment about it.